src/chigame/users/views.py

```diff
@@ -185,6 +185,25 @@ def cancel_friend_invitation(request, pk):
 
 
 @login_required
+def invite_to_game(request, pk, match_id):
+    sender = User.objects.get(pk=request.user.id)
+    receiver = User.objects.get(pk=pk)
```

Comment:
Everything should be in a try statement because there is no error handling for if the receiver or match don't exist. Same goes for tournament